setup.py

```diff
@@ -86,5 +115,6 @@ def get_packages():
     ],
     cmdclass={
         'install': Install,
+        'refresh_langs': RefreshLangs
```

Comment:
Does this require another command to be executed before key-mapper is fully installed? If yes, can this be added to `Install`?

Reply:
> Does this require another command to be executed before key-mapper is fully installed? If yes, can this be added to `Install` instead?

Yes, it requires user/admin to run `setup.py refresh_langs` when the translations are needed. I am a bit reluctant to put this into `Install` as the current approach gives more flexibility. After making a commit I found https://github.com/oleksis/youtube-dl-gui/blob/1dbc65d5fb7ca45038efaaf48259976eb4319099/setup.py#L267 where they're doing a similar thing. If you're still convinced this belongs to `Install` I will put it there ofc.

Reply:
quodlibet also uses `./setup.py build` as a separate command: https://quodlibet.readthedocs.io/en/latest/development/overview.html

lets call the command `build` as well

the new command should be part of
- the installation instructions for pip https://github.com/sezanzeb/key-mapper/blob/main/README.md
- the build script https://github.com/sezanzeb/key-mapper/blob/main/scripts/build.sh
- and the AUR package (I'll do that)

Reply:
I think after this and the mo folder this is probably ready to merge, thank you :)

Reply:
>     * the installation instructions for pip https://github.com/sezanzeb/key-mapper/blob/main/README.md

Sadly I've got no idea how can I instruct `pip` to run desired `setup.py` functions. Browsing tutorial did not reveal anything either :disappointed:

Reply:
either

```
git clone https://github.com/sezanzeb/key-mapper.git
cd key-mapper
python3 setup.py build
sudo pip3 install .
```

or put it into install after all to make `pip3 install git+https://github.com/sezanzeb/key-mapper.git` possible

Reply:
> or put it into install after all to make pip3 install git+https://github.com/sezanzeb/key-mapper.git possible

OK, I give up, will go this route.